lib/owlcarousel/owl.carousel.js

```diff
@@ -2895,7 +2895,7 @@
 		for (control in this._controls) {
 			this._controls[control].remove();
 		}
-		for (override in this.overides) {
+		for (override in this.overrides) {
```

Comment:
I believe that this should be `this._overrides` (leading underscore and two `r`s.